readme.md

```diff
@@ -257,13 +269,13 @@ After running the seeds, go into the `rails console` and play with the objects y
 
```

Comment:
Not letting me comment in the appropriate space but in this line " To run `db/seeds.rb` all we need to do is run `$ rails db:seed` in the Terminal." Get rid of the `$` before `rails db:seed`